gulp/util.js

```diff
@@ -33,6 +34,7 @@ exports.filterNonCodeFiles = filterNonCodeFiles;
 exports.readModuleArg = readModuleArg;
 exports.themeBuildStream = themeBuildStream;
 exports.minifyCss = minifyCss;
+exports.dedupeCss = dedupeCss;
```

Comment:
This seems to only be running for the full build of `angular-material.css` now, but I would also need it to run for the individual module css that it outputted from running `build-all-module` (specifically `gulp build-all-module --closure`).